sped_nfe/models/monta_nfe/inherited_sped_documento.py

```diff
@@ -94,7 +95,8 @@ def monta_nfe(self, processador=None):
         #
         # Duplicatas e pagamentos
         #
-        self._monta_nfe_cobranca(nfe.infNFe.cobr)
+        if self.condicao_pagamento_id.forma_pagamento == '14':
+            self._monta_nfe_cobranca(nfe.infNFe.cobr)
         self._monta_nfe_pagamentos(nfe.infNFe.pag)
```

Comment:
É para montar os dois?

Reply:
Podemos mudar o 14 por uma constante?

Reply:
As duplicatas são montadas somente quando a condição de pagamento for a Duplicatas Mercantil. Este código já foi modificado.